packages/suite/src/actions/wallet/coinjoinAccountActions.ts

```diff
@@ -49,84 +51,88 @@ const coinjoinAccountUnregister = (account: Account) =>
         account,
     } as const);
 
+const coinjoinAccountDiscoveryProgress = (account: Account, progress: ScanAccountProgress) =>
+    ({
+        type: COINJOIN.ACCOUNT_DISCOVERY_PROGRESS,
+        account,
+        progress,
```

Comment:
wrap it into `payload` for easier transformation to redux toolkit (and handle it in reducer)
```
payload: { account, progress }
```

i already [did the same](https://github.com/trezor/trezor-suite/pull/6566/commits/d5bea9d94436cf59e4b8229b8609cdd8bd6087d8) with other actions in separated PR

Reply:
Done.